src/feature/hs/hs_client.c

```diff
@@ -956,6 +956,27 @@ client_get_random_intro(const ed25519_public_key_t *service_pk)
   return ei;
 }
 
+/** Called when a rendezvous circuit has timed out. Every streams attached to
+ * the circuit will get set with the SOCKS5_HS_REND_FAILED (0xF3) extended
+ * error code so if the connection to the rendezvous point ends up not
+ * working, this code could be sent back as a reason. */
+static void
+socks_report_rend_circuit_timed_out(const origin_circuit_t *rend_circ)
```

Comment:
I'm not sure if 'report' is the right thing to call this. It's more of a 'mark' thing than a 'report', since by calling this function we dont report immediately.

Reply:
Replaced to `_mark_`. See fixup commit.